modules/kubernetes-addons/main.tf

```diff
@@ -122,6 +122,15 @@ module "cert_manager" {
   addon_context     = local.addon_context
 }
 
+module "aws_privateca_issuer" {
+  count             = var.enable_aws_privateca_issuer ? 1 : 0
+  source            = "./aws-privateca-issuer"
+  helm_config       = var.aws_privateca_issuer_helm_config
+  manage_via_gitops = var.argocd_manage_add_ons
+  irsa_iam_permissions_boundary = var.aws_privateca_issuer_irsa_permissions_boundary
```

Comment:
Can be removed from here and passed via `addon_context` as mentioned in the above comments.